### PR TITLE
Add a note for using http-only cameras

### PR DESCRIPTION
**Pull request by thinkl33t**
This tripped me up for about 3 days :grimacing:

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/camera.generic.markdown [hunk 1]
```diff
@@ -65,7 +65,7 @@ framerate:
   required: false
   type: integer
 verify_ssl:
-  description: Enable or disable SSL certificate verification.
+  description: Enable or disable SSL certificate verification.  Disable this to use a http-only camera or if you have a self-signed or invalid SSL certificate.
```
Comment (DubhAd) [inline source/_components/camera.generic.markdown]: _Set to `false` to use an http-only camera, or you have a self-signed SSL certificate and haven't installed the CA certificate to enable verification._

No need to mention invalid SSL certificates IMO

**Review by DubhAd — DISMISSED** [review]

File: source/_components/camera.generic.markdown (hunk 1 above)
Comment (thinkl33t) [reply source/_components/camera.generic.markdown]: Why not?  If people are using https to connect to 127.0.0.1:8123 they'll have one.

Comment (thinkl33t) [reply source/_components/camera.generic.markdown]: Oh, gotcha.  That works :)

**Comment by frenck** [comment]
The build was stuck.
In order to retrigger, I made some small grammar fixes on the page.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @thinkl33t! 👍

**Pull request merged**